utils/nctl/sh/scenarios/itst14.sh

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env bash
+
+source "$NCTL"/sh/scenarios/common/itst.sh
+
+# Exit if any of the commands fail.
+set -e
+
+#######################################
+# Runs an integration tests that tries to simulate
+# if a validator node can restart within single era 
+# and not equivocate.
+# Arguments:
+#   `timeout=XXX` timeout (in seconds) when syncing.
+#######################################
+function main() {
+    log "------------------------------------------------------------"
+    log "Starting Scenario: itst14"
+    log "------------------------------------------------------------"
+
+    # 0a. Verify all nodes are in sync
+    check_network_sync
+    log "Sleeping 10s"
+    sleep 10
+    check_network_sync
+    # 0b. Get era
+    STOPPED_ERA=$(check_current_era)
+    # 1. Stop node
+    do_stop_node "5"
+    # 2. Let the node go down
+    log_step "Sleeping for 10s"
+    sleep 10
+    # 3. Restart Node
+    do_read_lfb_hash 1
+    do_start_node "5" "$LFB_HASH"
+    # 4. Verify all nodes are in sync
+    check_network_sync
+    # 5. Verify network chain height sync
+    log "Sleeping 10s"
+    sleep 10
+    check_network_sync
+    # 6. Send deploy to stopped node
+    send_transfers '100'
```

Comment:
We shouldn't even need to send any transfers. Nodes will produce empty blocks. If we can simplify the test to its essential parts, I would prefer that.

Reply:
removed this function